recipe/meta.yaml

```diff
@@ -42,6 +44,8 @@ requirements:
     - python
     - {{ pin_compatible('llvmlite', max_pin='x.x') }}
     - {{ pin_compatible('numpy') }}
+    - llvmlite 0.27.* # otherwise conda test picks up llvmlite >= 0.28
+    - numpy 1.15.*
```

Comment:
Am curious why this line was needed (given we have `{{ pin_compatible('numpy') }}` above)? Does anyone remember? Also would it be possible to drop it?

Reply:
numpy 1.16 introduced NaT that makes time-related numba tests fail. I am not sure if the numba master is NaT aware.

Reply:
It looks like numba 0.43.1 has numpy 1.16 NaT support implemented.